lib/attribute.js

```diff
@@ -455,7 +456,10 @@ validators.format = function validateFormat (instance, schema, options, ctx) {
   if (typeof instance === 'undefined') {
```

Comment:
@shelbys I am wrong above, instance === undefined !! instance === null returns null which is true instead of false. Is this what is expected? false is only validate fails